### DISCUSSION TITLE
Recommendations for lightbox gallery

### DISCUSSION THREAD

**Discussion by branislavbrincko** [mantinedev/mantine#6376] [Q&A]
One thing I am missing from Mantine is a "lightbox gallery" component. A simple version could be implemented using a modal and carousel, but for more advanced features (like zoom, etc.), a library would be necessary. Could this be added to Mantine, or does anyone have a good recommendation for a library that would pair nicely with Mantine? Thanks in advance.

**Comment by rtivital** [comment]
Currently, Mantine does not provide such component, you can use a third-party package for that. Lightbox component might be added in one of the future releases.

**Comment by Toshinaki** [comment]
@branislavbrincko 
It' been a year, but recently I have basically the same needs for a lightbox / spotlight which you could click an image to open it and zoom, .etc.
So I searched but found only this discussion.
Then I created my own. If you're still interested: https://www.npmjs.com/package/@mantyke/spotlight-image
This is for only one image though (which is exactly what I want).

**Comment by rilrom** [comment]
For anyone who comes across this discussion, I created @mantine-bites/lightbox to handle all your lightbox needs.

Docs: https://rilrom.github.io/mantine-bites/lightbox/
Github: https://github.com/rilrom/mantine-bites/tree/main/packages/lightbox
